deployments/ansible/README.md

```diff
@@ -0,0 +1,57 @@
+# SignalFx Agent Ansible Playbook
+
+This is a Ansible playbook that will install and configure the SignalFx Agent.  To
+use it, simply include the role `signalfx-agent` in your play or run the playbook by passing the inventory.  That
+playbook defines the following parameters:
+
+ - `{{ conf }}` (Yaml): A config structure that gets directly converted to the agent
+    YAML.  See the [Agent Config
+    Schema](https://github.com/signalfx/signalfx-agent/blob/master/docs/config-schema.md)
+    for a full list of acceptable options.  The only required option is
+    `signalFxAccessToken`.  Here is a basic config that will monitor a basic set of
+    host-level components:
+    
+    ```yaml
+    conf:
+      signalFxAccessToken:
+      monitors:
+        - type: collectd/cpu
+        - type: collectd/cpufreq
+        - type: collectd/df
+        - type: collectd/disk
+        - type: collectd/interface
+        - type: collectd/load
+        - type: collectd/memory
+        - type: collectd/protocols
+        - type: collectd/signalfx-metadata
+        - type: collectd/uptime
+        - type: collectd/vmem
+    ```
+
+	It is probably going to be simpler to keep this config in `target-group` var file at the 
+	`group_vars` and update the same in playbook, which will make it visible to that group of hosts under inventory.
+
+ - `{{ package_stage }}`: The package repo stage to use: `main`, `beta`, or `test`
```

Comment:
"main" was switched to "final" before the initial release.

Reply:
Updated